ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -569,7 +572,7 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
                                     onChange={value => this.updateAddress('state', value)}/>
               </FlexRow>
               <FlexRow style={{marginTop: '0.75rem'}}>
-                <TextInputWithLabel dataTestId='zip' inputName='zip' placeholder='Zip Code'
+                <TextInputWithLabel dataTestId='zip' inputName='zip' placeholder='Zip code'
                                     value={zipCode} labelText='Zip Code'
```

Comment:
Can you change the labelText as well if the placeholder changes? Should you?

Reply:
Yes! I should. That is the one that actually changes the display text. Thanks for finding that.